application/cli-app/src/main/java/org/mobilitydata/gtfsvalidator/Main.java

```diff
@@ -22,7 +22,8 @@
 import org.mobilitydata.gtfsvalidator.config.DefaultConfig;
 import org.mobilitydata.gtfsvalidator.usecase.ParseSingleRowForFile;
 
-import java.io.*;
```

Comment:
Looks like there is some shuffling of imports here (i.e., this commit changes the imports committed by someone else), which usually means that we collectively aren't using the same IntelliJ formatting template. Or someone doesn't have "Optimize imports" checked when committing within IntelliJ. It would be good to standardized on a code format template.

Reply:
I will check this with @fabrice-v

Reply:
We will check the `Optimize import` in the commit dialog box, in addition to `Reformat code`
![image](https://user-images.githubusercontent.com/55884852/76627665-c8723a00-6511-11ea-9b99-d05054794669.png)

Reply:
:+1: FWIW, you can enable this as a default in IntelliJ via "File->Settings". Just search for "import" and it's under the version control section